jwst/datamodels/schemas/filteroffset.schema.yaml

```diff
@@ -38,3 +40,5 @@ allOf:
             type: number
           column_offset:
             type: number
+	
```

Comment:
Looks like these are bogus lines that should be removed. I think they're also responsible for killing the CI tests, because of the tab character on each of those 2 lines.